pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardbasic/_http.log

```diff
@@ -23,26 +25,28 @@ X-Xss-Protection: 0
 
 ---
 
-POST https://monitoring.googleapis.com/v1/projects/${projectId}/dashboards?alt=json
+POST https://monitoring.googleapis.com/v1/projects/${projectId}/dashboards?%24alt=json%3Benum-encoding%3Dint
 Content-Type: application/json
-User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
+User-Agent: kcc/controller-manager
+x-goog-api-client: gl-go/1.22.3 gapic/1.19.0 gax/2.12.3 rest/UNKNOWN
+x-goog-request-params: parent=projects%2F${projectId}
 
 {
   "columnLayout": {
     "columns": [
       {
-        "weight": 2,
+        "weight": "2",
         "widgets": [
           {
             "title": "Widget 1",
             "xyChart": {
               "dataSets": [
                 {
-                  "plotType": "LINE",
+                  "plotType": 1,
```

Comment:
Kind of annoying, but the official client uses integer enum encoding, messing up our diffs.  I can try the proto form and see if it is any better.

Reply:
I missed this message. Please ignore my other comment.